Fix Communication Readme and Sample issues (#18977)
Azure/azure-sdk-for-net: sdk/communication/Azure.Communication.Common/README.md

### Create a credential with a static token

> For a short-lived clients when refreshing token upon expiry is not needed, `CommunicationTokenCredential` can be instantited with a static token.

Reviewer: ```suggestion
For a short-lived clients, refreshing the token upon expiry is not necessary and `CommunicationTokenCredential` may be instantiated with a static token.
```
